modules/ossm-config-otel.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/traces/ossm-distr-tracing-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-otel_{context}"]
+= Configuring {OTELName} with Service Mesh
```

Comment:
So, this is not the same as the topic mapm _Using distributed tracing with Service Mesh_, which I found confusing when I went to the page. IMHO these should be the same title.

Reply:
It will likely all change for GA as more content is added.